x-pack/filebeat/input/netflow/doc.go

```diff
@@ -5,3 +5,6 @@
 package netflow
 
 //go:generate go run fields_gen.go -output _meta/fields.yml --column-name=2 --column-type=3 --header _meta/fields.header.yml decoder/fields/ipfix-information-elements.csv
+//go:generate go run fields_gen.go -output _meta/fields.yml --append --column-name=3 --column-type=4 --header _meta/fields.header.yml decoder/fields/cert_pen6871.csv
```

Comment:
Looks like this approach of appending won't work b/c we need to merge the fields together to avoid duplicates.